packages/satyrographos/satyrographos.0.0.2.1/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "SAKAMOTO Noriaki <mrty.ityt.pt@gmail.com>"
+authors: [
+  "SAKAMOTO Noriaki <mrty.ityt.pt@gmail.com>"
+]
+homepage: "https://github.com/na4zagin3/satyrographos"
+dev-repo: "git+https://github.com/na4zagin3/satyrographos.git"
+bug-reports: "https://github.com/na4zagin3/satyrographos/issues"
+license: "LGPL3+"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "core" {< "v0.13"}
+  "dune" {build}
```

Comment:
```suggestion
  "dune" {>= "1.4"}
```
Dune shouldn't be tagged as a build dependency (see #14266). Furthermore, the `dune-project` file in your repository says this package requires at least dune 1.4.

Reply:
Thank you for letting me know that!